chat/management/commands/rundiscordbot.py

```diff
@@ -61,36 +63,43 @@ async def send_puzzles_stuck(message):
         Puzzle.objects.filter(
             Q(hunt=settings.BOT_ACTIVE_HUNT),
             Q(status=Puzzle.STUCK) | Q(status=Puzzle.EXTRACTION),
-        )
+        ).select_related("chat_room")
     )
     await send_puzzles(message, puzzles, "Stuck puzzles")
 
 
 async def send_puzzles(message, puzzles, title):
     print(f"Sending puzzles {puzzles}")
-    embed = discord.Embed()
+    embed = discord.Embed(title=title)
     lines = []
     for p in puzzles:
-        line = "- "
+        line_title = ""
         if p.is_solved():
-            line += f"[{p.answer}] "
-        line += f"[{p.name}]({p.url})" if p.url else p.name
+            line_title += f"[{p.answer}] "
+        line_title += p.name
+
+        line = ""
+        if p.url:
+            line += f"[Puzzle]({p.url}) "
         if p.sheet:
-            line += f" ([sheet]({p.sheet}))"
-        lines.append(line)
+            line += f"([sheet](https://smallboard.app/puzzles/s/{p.id}))"
+        if p.chat_room:
+            if p.chat_room.text_channel_url:
+                line += f"([chat]({p.chat_room.text_channel_url}))"
+
+        lines.append((line_title, line))
     print(f"lines: {lines}")
-    chunk_lines = []
-    chunk_length = 0
+    lines.sort()
+    field_count = 0
     for line in lines:
-        line_length = len(line)
-        if (chunk_length + line_length) >= 1024:
-            embed.add_field(name=title, value="\n".join(chunk_lines))
-            chunk_lines = []
-            chunk_length = 0
-        chunk_lines.append(line)
-        chunk_length += line_length
-    if chunk_lines:
-        embed.add_field(name=title, value="\n".join(chunk_lines))
+        line_length = sum([len(x) for x in line])
+        if (line_length + len(embed)) >= 6000 or field_count >= 25:
```

Comment:
If available, can we leave a comment linking to the docs where these limits are mentioned? (I know I didn't include one in the original PR, but it seems nice to have now that I look at it again)